lib/manageiq/automation_engine/logger.rb

```diff
@@ -28,31 +42,37 @@ class Logger < ManageIQ::Loggers::Base
 
       def info(progname = nil, resource_id: nil, &block)
         severity, message, progname = add_to_db(INFO, nil, progname, resource_id: resource_id, &block)
+        wrapped_logger.add(severity, message, progname, &block)
         add(severity, message, progname, &block)
```

Comment:
I think you only need the wrapped logger? (or does this allow writing to a passed in logdev - not sure why we need that)

Reply:
Oh test - interesting - wondering if we need that at all, and just pass an alternative wrapped_logger

Reply:
yes, it's for test alone to keep the changes minimal